cloud/resource_cloud_environment.go

```diff
@@ -221,10 +244,68 @@ func resourceCloudEnvironmentDelete(ctx context.Context, d *schema.ResourceData,
 	}
 	namespace := d.Get("organization").(string)
 	name := d.Get("name").(string)
+	waitForCompletion := d.Get("wait_for_completion")
 	err = clientSet.CloudV1alpha1().CloudEnvironments(namespace).Delete(ctx, name, metav1.DeleteOptions{})
 	if err != nil {
 		return diag.FromErr(fmt.Errorf("DELETE_CLOUD_ENVIRONMENT: %w", err))
 	}
+
+	_, err = clientSet.CloudV1alpha1().CloudEnvironments(namespace).Get(ctx, name, metav1.GetOptions{})
+	if err != nil {
+		return diag.FromErr(fmt.Errorf("ERROR_READ_CLOUD_ENVIRONMENT: %w", err))
+	}
```

Comment:
I think we don't need this since retry can cover this, no?

Reply:
Correct, I removed it